source/utils/formatDateToMMdd.mc

```diff
@@ -0,0 +1,13 @@
+import Toybox.Lang;
+import Toybox.Time;
+import Toybox.Time.Gregorian;
+
+function formatDateToMMdd(time as Time.Moment) as Array<String> {
```

Comment:
There is a problem here which is already solved in the `formatDateToEng` function. The thing is that `date.month.toString()` can return a string in different languages, which is not supported by the font we use, which leads to a broken dial, for this we will have to add localization and change fonts. So for now I recommend to do the following, we need to do a small refactoring of the `formatDateToEng` function and make it reusable in your function. I would do it this way, create a `formatDate` function that would return an array with day and month, `formatDateToEng` would take the values from it and concatenate them into a string in `DD MM` format, your function would do the same thing but concatenate the data into a string as `DD MM`.